korean/explore-outside/find-coco.md

```diff
@@ -0,0 +1,2 @@
+아 근데 같이 먹을 마쉬멜로우를 가져온다는걸 깜빡했네.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.